tests/tests/spec_yes_env_no_with_jfac_fail1/spec_yes_env_no_with_jfac_fail1.out.cmp

```diff
@@ -128,6 +128,7 @@ AtChem2 v1.3-dev
  6      BLHEIGHT              NOTUSED
  7      DILUTE                NOTUSED
  8      JFAC                       J4
+STOP 2
```

Comment:
Can this be correct? A `STOP` being printed, then further output afterwards? Seems odd.

Reply:
To be honest I do not know! The test is meant to fail, and it seems that `STOP` is being printed to screen at least on my Linux machine. It seems it is not necessary for the jobs that run on actions. A bit puzzled.

Reply:
The difference between the two is probably something to do with how the two streams (output and error) are being handled, either piped to the same location or different ones. The main question is does it pass/fail the tests correctly I guess :)

Reply:
This particular test passes. It is four others that fail but only on mac.